modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,87 @@
+
+== Policy Injection and Enforcement
+
+When you deploy an API proxy application for a specific API version to a Mule Runtime, any applied policies in the platform, or offline policies in the runtime, are injected into the API proxy. This changes the behavior of the application. When the proxy application receives a new request, all injected policies are applied to decide if, and how, the request is forwarded to the API implementation endpoint.
+
+In this way, the actual policy enforcement occurs inside the proxy application itself, minimizing the cross talk between the API proxy, which is processing the received request, the Anypoint Platform agent, and online API Manager. The API proxy application does not require any communication with the Anypoint Platform agent running in the runtime, nor does it require any communication with API Manager.
+
+However, the Anypoint Platform agent remains connected to API Manager. After policies are reconfigured or removed from API Manager, those policies are downloaded to any connected API Gateway or Mule runtimes, which updates each runtime /policies folder. The policy changes are then again injected into each API proxy application. This allows policies to be dynamically changed without having to redeploy the API proxy application, and without having to restart the API Gateway or Mule runtimes.
+
+
+
+== Policy Scope
+
+Policies are isolated from the application and other policies. However, there are ways to expose information from a policy. The authentication information about a policy can be propagated in the Security Context Principal object.
+
+Some policies, such as the token enforcement policy, allow propagation of headers, including the information related to the token. This token is used by the implementation service or by the API itself.
+
+//Eva--add info about the generic policy scope Add an example or link here..Fede Balbi?
+
+
+== Policy Size
+
+The size of a policy varies based on its dependencies. The first time Mule Runtime polls for policies it will take longer to fetch them. However, after policies are retrieved, they are cached on the file system. This ensures offline availability and also reduces latency to fetch policies.
+//Add more information about the policy size. we have two options here, either hide the policy size, or clarify that this doesn't happen for cloudhub or runtime fabric--chirag
+
+
+
+== Policy Level
+
+Policies implement an additional level of granularity, where access is controlled based on a criteria. In this level of granularity, policies are applied to only those request that match the criteria.  These polices are known as resource-level policies. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+Resource level policy support includes RAML and HTTP-based APIs (since Mule 4). A policy applied at a resource level affects all HTTP methods, for example, PUT, GET, POST, or selected HTTP methods, within the resource.
```

Comment:
```suggestion
Starting with Mule 4, resource-level policy support includes RAML and HTTP-based APIs. A policy applied at a resource level affects all HTTP methods within the resource: PUT, GET, POST, or selected HTTP methods.
```

Reply:
Same comment as above, is it clear that resource level is also supported in Mule 3 for RAML based APIs?

Reply:
Yes, Fixed it now! Done! Thanks!